src/cms/static/js/forms/update-permalink.js

```diff
@@ -0,0 +1,25 @@
+/* this file updates the permalink of the current page form
+when the user edited the page translations title */
+
+u("#id_title").on('focusout', function(e){
```

Comment:
Oh, and one additional comment: Since umbrellaJS is deprecated and will be removed in PR #731, it probably makes sense to use the standard JS-syntax already. Sorry, that you had to learn a new syntax which is now being discarded :unamused: 
```suggestion
document.getElementById("id_title").addEventListener("focusout", function(e){
```

Reply:
I remember, you said that recently during a call, so using umbrella here is not "future-proof" :sweat_smile:
I will update to standard js, that's not much effort for that PR, but it was fun working with umbrella.
Umbrella has so many common features to jquery, I think gaining experience in this area is an advantage :smile: